frontend/front/src/components/SurveyComponent/HeatPumpDropdown.js

```diff
@@ -21,6 +21,21 @@ export const HeatPumpDropdown = ({
 }) => {
   const { field: groupField, formState } = useController({ name, control });
 
+  const labelFormRef = useRef(null)
+  const [_, setResizeTrigger] = useState(0)
+
+  useEffect(() => {
+    const refresh = () => setResizeTrigger(prev => prev+1)
+    
+    refresh()
+
+    window.addEventListener('resize', refresh)
+      
+    return () => window.removeEventListener('resize', refresh)
+
+  })
```

Comment:
Check the linting test, this needs a dependency array even if it's just `[]`.

Using an event listener and a useState to trigger a refresh here seems like a red flag to me. Would it be possible to have the useEffect depend on `labelFormRef.current?.scrollHeight` at least? I've looked into this 0% and this may be the only way but it seems like there ought to be a purely CSS way to handle this sort of thing.